NuGet.Config

```diff
@@ -4,5 +4,6 @@
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
     <add key="NuGet" value="https://nuget.org/api/v2/" />
     <add key="AzureAD" value="http://www.myget.org/F/azureadwebstacknightly"/>
+    <add key="aspnet-contrib" value="https://www.myget.org/F/aspnet-contrib"/>
```

Comment:
Meh, removing it from this PR will be easier. Just tell me we're done with this PR, and I'll revert these changes.